containers/ddev-webserver/ddev-webserver-base-files/etc/apache2/apache2.conf

```diff
@@ -185,6 +185,24 @@ AccessFileName .htaccess
 </FilesMatch>
 
 
+# Security: Enforce file types matching at end of filename only
```

Comment:
Is it OK that this affects all project types?

Reply:
I'm not 100% sure but this should normally work with all project types. If a project relays on this unsecure default configuration it should be solved there directly. This change makes sure the proper file extension is detected means the last .*